server/game/cards/10-TEL/VoiceOfTheAncestors.js

```diff
@@ -44,8 +44,8 @@ class VoiceOfTheAncestors extends DrawCard {
                             AbilityDsl.effects.gainAbility(AbilityTypes.Persistent, {
                                 match: (card, context) => card === context.source.parent,
                                 effect: [
-                                    AbilityDsl.effects.modifyMilitarySkill((card, context) => context.source.printedMilitarySkill || 0),
-                                    AbilityDsl.effects.modifyPoliticalSkill((card, context) => context.source.printedPoliticalSkill || 0)
+                                    AbilityDsl.effects.attachmentMilitarySkillModifier((card, context) => context.source.printedMilitarySkill || 0),
+                                    AbilityDsl.effects.attachmentPoliticalSkillModifier((card, context) => context.source.printedPoliticalSkill || 0)
```

Comment:
This depends on your ruling, but I think the bonuses provided here are more like Katana of Fire

Reply:
Agreed, willl revert